lib/spritely/collection.rb

```diff
@@ -19,7 +19,7 @@ def find(name)
     end
 
     def cache_key
-      files.collect { |file| Digest::MD5.file(file) }.join
+      files.flat_map { |file| [Digest::MD5.hexdigest(file), Digest::MD5.file(file)] }.join
```

Comment:
I think naming the block param `file_path` would make what's going on here a bit clearer.